src/components/DateTimeList/DateTimeList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import DateTimeListElement from '../DateTimeListElement';
+
+export default function DateTimeList(props) {
+  const { dateTimeList, setDateTimeList } = props;
+
+  const addDateTime = (e) => {
+    e.preventDefault();
+    setDateTimeList([...dateTimeList, { id: Math.max(...dateTimeList.map((item) => item.id)) + 1, time: '17:30' }]);
```

Comment:
@uriklar this is a bit risky. maybe use `short-uuid` ?

Reply:
do you need this id for firebase or react (key for map) @uriklar ?

Reply:
Not really.. I can key by the date+time. Itay added it. I think we can remove it

Reply:
Whatever you decide

Reply:
lets's leave for now